### ISSUE TITLE
Is the Interval interface needed?

### ISSUE THREAD

**Issue by tidoust** [webtiming/timingobject#12]
The specification defines a `TimingInterval` interface [1], which provides a clean way to define range restrictions on the position of a `TimingStateVector` along its unidimensional axis.

However, while HTML5.1 has a similar notion of range, in the constructor of `DataCue` in particular [1], it does not define a specific interface for that. Also, the `lowInclude` and `highInclude` attributes are not used in the Timing Object specification so far.

In short, I think that the `Interval` interface should either be clearly motivated and the spec updated accordingly, or replaced by `startTime` and `endTime` attributes.

[1] http://webtiming.github.io/timingobject/#dfn-interval
[2] http://www.w3.org/TR/html51/semantics.html#datacue

**Comment by ingararntzen** [comment]
Agree. The approach with timedtexttrack aligning with existing texttrack spec removes the motivation for this. Please remove it.

The reason I introduced this in the Sequencer implementation was to allow cues to be laid out back-to-back  (e.g.  cue1: [a,b], cue2: [b,c] and cover the entire "timeline" without introducing ambiguity concerning possible overlap at shared endpoints (i.e. point b) --- is both cue1 and cue2 valid in b or just one of them?  Avoiding this ambiguity by defining intervals like [a,b> and [b,c> makes it easier for the programmer and also allows him/her to exploit precedence rules regarding event ordering.

**Comment by tidoust** [comment]
OK, I dropped the definition of Interval, now replaced with `startPosition` and `endPosition` attributes. Closing this issue as a result.
